<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>rep.rate[MHz]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <double>120.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="toolTip">
         <string>To read one-column TCSPC-data check this box. The one column-data is expected to be in the order VV, VH. If this is checked a dt has to be specified!</string>
        </property>
        <property name="text">
         <string>VV/VH-Jordi</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="toolTip">
         <string>If this is selected the time-axis is multiplied by the dt specified in the spin-box.</string>
        </property>
        <property name="text">
         <string>dt[ns]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>120.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
        <property name="value">
         <double>0.014100000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="8">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_3">
           <property name="toolTip">
            <string>If this is selected the first part of the one-column data is read</string>
           </property>
           <property name="text">
            <string>VV</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="toolTip">
            <string>If this is selected the second part of the one-column data is read</string>
           </property>
           <property name="text">
            <string>VH</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>VM</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="toolButton">
           <property name="text">
            <string>shift</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>rebin</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionDtChanged">
   <property name="text">
    <string>dtChanged</string>
   </property>
  </action>
  <action name="actionRebinChanged">
   <property name="text">
    <string>RebinChanged</string>
   </property>
  </action>
  <action name="actionRepratechange">
   <property name="text">
    <string>repratechange</string>
   </property>
  </action>
  <action name="actionPolarizationChange">
   <property name="text">
    <string>PolarizationChange</string>
   </property>
  </action>
  <action name="actionGfactorChanged">
   <property name="text">
    <string>gfactorChanged</string>
   </property>
  </action>
  <action name="actionIsjordiChanged">
   <property name="text">
    <string>isjordiChanged</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionDtChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>actionDtChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionRebinChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>actionRepratechange</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionPolarizationChange</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionPolarizationChange</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_3</sender>
   <signal>valueChanged(double)</signal>
   <receiver>actionGfactorChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionIsjordiChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
